modules/AOP_Case_Updates/language/en_us.lang.php

```diff
@@ -57,6 +57,7 @@
   'LBL_AOP_CASE_UPDATES_SUBPANEL_TITLE' => 'Case Updates',
   'LBL_CASE_NAME' => 'Case',
   'LBL_CONTACT_NAME' => 'Contact',
-  'LBL_INTERNAL' => 'Internal Update',    
+  'LBL_INTERNAL' => 'Internal Update',
```

Comment:
This PR, on file modules/AOP_Case_Updates/Case_Updates.php
included 2 additional strings but they are not here. Are the translations pulled from other file?

+    $saveBtn = $app_strings['LBL_SAVE_BUTTON_LABEL'];
+    $saveTitle = $app_strings['LBL_SAVE_BUTTON_TITLE'];